app/uk/gov/hmrc/gform/gform/CryptoUtil.scala

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.gform.gform
+
+object CryptoUtil {
+
+  def sha256Hash(text: String): String =
+    String.format(
+      "%064x",
+      new java.math.BigInteger(
+        1,
+        java.security.MessageDigest
+          .getInstance("SHA-256")
+          .digest(
+            text
+              .getBytes("UTF-8")
+          )
+      )
+    )
```

Comment:
I think we can simplify this:

```suggestion
import java.nio.charset.StandardCharsets

def sha256Hash(text: String): String =
    java.security.MessageDigest
      .getInstance("SHA-256")
      .digest(
        text
          .getBytes(StandardCharsets.UTF_8)
      )
      .map("%02x".format(_))
      .mkString
```